src/apps/viewer/index.html

```diff
@@ -38,6 +38,19 @@
     </head>
     <body>
         <div id="app"></div>
+        <div style="position: absolute; left: 100px; top: 5px; width: 120px; height: 100px;">
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, '1tqn', 'metalc1');">metalc1</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, '1tqn', 'metalc2');">metalc2</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, '5elb', 'covale3');">covale3</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, undefined, 'covale4');">covale4</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, undefined, 'covale5');">covale5</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, undefined, 'covale6');">covale6</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, undefined, 'disulf1');">disulf1</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, undefined, 'disulf4');">disulf4</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, undefined, 'disulf6');">disulf6</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.inspectStructConn(molstarViewer.plugin, undefined, 'disulf9');">disulf9</button>
+            <button onclick="molstarViewer?.plugin && molstar.ExtensionData.wwPDBStructConn.clearStructConnInspections(molstarViewer.plugin, undefined);">Clear</button>
+        </div>
```

Comment:
Please remove this. The default viewer instance is no place to showcase extensions like this.

The example in the .md file will have to be enough for now.

Reply:
Oh, sorry, I forgot to remove it.